dask/bag/text.py

```diff
@@ -106,8 +106,10 @@ def read_text(
         )
         if files_per_partition is None:
             blocks = [
-                delayed(partial(file_to_blocks, include_path, delimiter=linedelimiter))(
-                    fil
+                delayed(list)(
+                    delayed(
```

Comment:
I suppose we should explicitly test the output of this branch, one list (len files) of lists (blocks per file) of text.

Reply:
This was a change introduced accidentally in#7349 I think.

Reply:
Maybe, but it didn't fail anything right away - there should be a test for `db.read_text(fn, collection=False)`

Reply:
I added that in this PR.